src/robot/libdocpkg/model.py

```diff
@@ -74,21 +100,45 @@ def save(self, output=None, format='HTML'):
         with LibdocOutput(output, format) as outfile:
             LibdocWriter(format).write(self, outfile)
 
+    def convert_doc_to_html(self):
+        formatter = DocFormatter(self.keywords, self.doc, self.doc_format)
+        self._doc = formatter.html(self.doc, intro=True)
+        self.doc_format = 'HTML'
+        for init in self.inits:
+            init.doc = formatter.html(init.doc)
+        for keyword in self.keywords:
+            keyword.doc = formatter.html(keyword.doc)
+
```

Comment:
With that "builtin" html doc converter it is much easier to generate html containing docs.
XmlWriter gets easier and HTMLWriter as well